Exequiel#29

Exequielac opened the pull request:
Revisar todo lo relacionado con los issues #27 #28 y #29 . Prestar atención especial al apartado de cambios locales de iluminación (local illumination changes) porque puedo haber metido la pata. Ya queda poco :D.

Commit:
Definición de la técnica realizada #29

Commit:
Añadidas funciones para la costura de imágenes y definición de algunos puntos a ampliar #29

Commit:
Refinamiento de la descripción de la técnica #29

Commit:
Añadida imagen del artículo para seamless tiling

Commit:
Ejemplo de aplicación de la técnica hecho

Commit:
Añadida descripción de los resultados obtenidos #29

Commit:
Visualización de resultados de costuras #29

Commit:
Realizada interpretación de los resultados

Commit:
Añadida imagen de golosinas #29

Commit:
Terminado ejemplo de las golosinas #29

Commit:
Sustituido ejemplo de golosinas por madera y añadido ejemplo de nubes #29

Commit:
Seamless tiling terminado #29

Commit:
Subo Local Illumination Changes en sucio, mañana lo pongo bonico y lo refino

Commit:
Termino la parte de local color changes, mañana lo refino y lo pongo bonico

BlancaCC committed:
El borrado de la pintada ya funciona #25

- Se ha experimentado con pegado directo de dos regiones.
- Con ir concatenando el resultado de una interpolación.

Falta:
- Redactar conclusiones
- Experimentar con clonar el grafiti en una pared y restar la máscara que se aplica en el otro sitio jejejje

BlancaCC committed:
Añado algunas imágenes de paredes selecciones y trazos #25

Quizás habría que borrar alguna que no se utlice, dejo para el final la limpieza

Commit:
Añadida descripción y comentarios de local illumination changes

Commit:
Terminada experimentación con cambios de iluminación locales #27

BlancaCC committed:
El experimento sobre quitar máscara inversa ha sido un éxito #25

Sea I_0 la imagen original con la pintada a borrar.
La idea consiste en clonar la pintada que se quiere borrar  a un lado sin pintada de la pared aplicando el movimiento rígido T, obtenendo con ello la imagen I_1
Sea M la  diferencia I_1 - I_0, M es una máscara que filosóficamente significa que hay que añadir a la pared para que aparezca la pintada.

Si hacemos I_0 - T^{-1} M se obtendrá la imagen con el grafiti borrado :D

Falta hacer la memoria de esta parte 😚

BlancaCC committed:
Añado nueva imagen #25

Commit:
Terminado cambios de iluminación locales #27

BlancaCC committed:
Redacto explicación del algoritmo nuevo #25

BlancaCC committed:
Termino de escribir conclusión de #25 Falta revisión

Commit:
Terminado cambios de color locales #28

Commit:
Pequeña correción en cambios de color locales #28

Commit:
Faltaba pequeño comentario de cambios locales de color #28

Commit:
Mergeado de main con #27 #28 y #29

BlancaCC commented:
Hola, analizando el apartado de costura de imágenes y hay una cosa que no termino de entender muy bien, el objetivo de la figura 86.  La idea, según percibo es ofrecer un ejemplo de que el cosido no es efectivo, sin embargo el ejemplo de la nube me parece desafortunado por ser muy dispar, siendo quizás demasiado de sentido común que falle. 

¿has probado esa misma imagen recortada de tal forma que no sea azul? ¿Cuál podría ser el objetivo de coser una imagen tan diferente como el cielo azul.

Exequielac commented:
@BlancaCC Este ejemplo lo puse como réplica de otro proyecto que vi y la conclusión que sacaban era que el algoritmo no es perfecto (era evidente) y que la selección de la imagen inicial debe de realizarse con cuidado

BlancaCC committed:
[costura] Agrego introducción y amplio conclusión añadiendo generalizaciones de la técnica #29

La introducción la he añadido para que se entienda un poco más donde van las intenciones, porque no ha sido hasta
el final de varias lecturas que he entendido un poco qué era una costura y tal jejjeje

La generalización,diría  que se me ha ido un poco la cabeza, sobretodo en el apartado de determianción de la costura

BlancaCC committed:
Cambio subtitulo a determinación de la costura en conclusiones #29 #50

BlancaCC commented:
Respuesta a TODO: ¿Se debería de referenciar al paper Fattal et al. 2002? Lo he mirado y es bastante técnico.

Por supuesto que debería hacerse y más mencionándolo como haces al principio. 
Lo dejo que lo arregles ya que no tengo ni idea del artículo que es 🤣

Commit:
Añadida referencia de Fattal et al. #27

Commit:
Rama mergeada con blanca#25

BlancaCC committed:
Reviso y cambio minucias de #27 #50
Muy buen trabajo Exequiel 🥰

BlancaCC committed:
[Cambio de colores #28 ] Añado experimento para aumentar la gama cromática

Se basa en conocer la debilidad del algoritmo en caunto al valor de los canales y permutarla con otro
Falta redactar las nueva conclusión y hablar del valor que pueden tomar los coeficientes.

BlancaCC committed:
Termino de redactar conclusiones de cambio de colores #28
y reviso #27 y #29

Puesto que he cambiado cosillas de 28 por favor vuélvelo a revisar y ya mergea
